terraform/environments/delius-mis/locals_development.tf

```diff
@@ -4,8 +4,8 @@ locals {
   environment_config_dev = {
     legacy_engineering_vpc_cidr            = "10.161.98.0/25"
     legacy_counterpart_vpc_cidr            = "10.162.32.0/20"
-    legacy_ad_domain_name                  = "delius-mis-dev.local"
-    legacy_ad_ip_list                      = ["10.162.36.235", "10.162.35.251"]
+    ad_domain_name                         = "delius-mis-dev.internal"
```

Comment:
Correct me if I'm wrong, but should be able to get the IP list from `aws_directory_service_directory.mis_ad.dns_ip_addresses` instead of hard coding. Worth a try.